pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -78,3 +78,17 @@ type RemoveDuplicatesArgs struct {
 	Namespaces        *api.Namespaces
 	ExcludeOwnerKinds []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// RemovePodsViolatingTopologySpreadConstraintArgs holds arguments used to configure RemovePodsViolatingTopologySpreadConstraint plugin.
+type RemovePodsViolatingTopologySpreadConstraintArgs struct {
+	metav1.TypeMeta
+
+	Namespaces                 *api.Namespaces
+	LabelSelector              *metav1.LabelSelector
+	IncludeSoftConstraints     bool
+	ThresholdPriority          *int32
+	ThresholdPriorityClassName string
```

Comment:
```suggestion
````
This is now done at a higher level.
Validation is now checked [here](https://github.com/kubernetes-sigs/descheduler/blob/83c4f5d5261e7113b6367c70914cedd1af7792ff/pkg/descheduler/descheduler.go#L371-L379)

And it's added to EvictionFilter [here](https://github.com/kubernetes-sigs/descheduler/blob/83c4f5d5261e7113b6367c70914cedd1af7792ff/pkg/descheduler/descheduler.go#L389)